cmd/aries-js-worker/index.html

```diff
@@ -13,22 +13,39 @@
     <meta charset="utf-8"/>
     <script src="dist/web/aries.js"></script>
     <script>
-        // TODO fix the stutter in the API's exports ("Aries.Aries....")
-        const aries = Aries.Aries({assetsPath: "/dist/assets"})
+        var aries
         var stopNotifier
 
-        async function handleStart() {
+        async function startAries() {
             const request = document.querySelector("#opts").value
-            const response = await aries.start(request)
-            document.querySelector("#output").value = JSON.stringify(response)
+
+            // TODO fix the stutter in the API's exports ("Aries.Aries....")
+            aries = Aries.Aries({assetsPath: "/dist/assets", startOpts: request})
```

Comment:
we don’t await like we did with start?

Reply:
No, destroy function doesn't return promise, instead it will wait for promise(`aries.stop()`) to terminate worker.

Reply:
I am going to update upcoming patch where it constructor returns promise. Then will put await here.

Reply:
constructor to return promise, and added await.